README.md

```diff
@@ -1 +1,18 @@
-# hw01
\ No newline at end of file
+# hw01
+ 
+## **All about Whitney B**
+![alt text](https://github.com/whitneybeamer/hw01/blob/master/IMG_4622-1.JPG)
```

Comment:
Go you for figuring out how to link a photo through your github. I wasn't able to figure that out, so I ended up using imgur and uploading and linking my photo from there,

Reply:
Alternatively, you can use the shortened path to embed the image

```
![alt text](IMG_4622-1.JPG)
```

Since `README.md` and `IMG_4622-1.JPG` are in the same folder, this will link directly to the image, even if someone else were to later fork your repo